docs/cpan-dependency-confusion.md

```diff
@@ -0,0 +1,239 @@
+# CPAN dependency confusion
+## Rationale 
+Take advantage of an incorrectly configured setup that resolves a public module (published on CPAN) in place of the expected private/local version due to one of the following reasons:
+- Local module is not installed at the time of running the CPAN client
+- Public CPAN version is preferred to local
+- Using a CPAN repository (proxy) that can be confused
+
+This type of vulnerability was proved to affect other ecosystems (PyPI, RubyGems and NPM). This is presented in [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) by [Alex Birsan](https://medium.com/@alex.birsan).
+
+## Background
+"Dependency confusion" takes advantage of mixing (semi-)private modules with public CPAN modules.
+
+In the Perl ecosystem, it could be one or more of the following options:
+- Using a "DarkPAN" along with CPAN
+- Using a "GrayPAN" along with CPAN
+- Mixing manual installs (`make install`, "blib install"...) along with CPAN
+
+See [About the various PANs](https://www.olafalders.com/2019/02/19/about-the-various-pans/) for details about DarkPAN and GrayPAN.
+
+An attacker can "claim" a public namespace, install a high version (to take precedence in the index over a smaller local version number) and wait for vulnerable setups to install (and execute) their newly uploaded module.
+
```

Comment:
Add a discussion about how PAUSE and MetaCPAN highlight unauthorised and TRIAL releases.

Reply:
Yep, gonna do that

Reply:
Added